desktop-app/src/renderer/store/features/renderer/index.ts

```diff
@@ -116,6 +124,13 @@ export const rendererSlice = createSlice({
     setRotate: (state, action: PayloadAction<boolean>) => {
       state.rotate = action.payload;
     },
+    setCss: (state, action: PayloadAction<InjectedCss | undefined>) => {
```

Comment:
Instead of keeping the injected state in redux, we can keep the selected test mode so that we can show an indication in the UI (like a tick mark near the option name) on which simulation is currently on.

Reply:
Ready for review, I will leave the change from redux to local component state to you.

Reply:
I went ahead and made the changes to move redux store to local component state.